.gitlab-ci.yml

```diff
@@ -28,12 +28,22 @@ stages:
     - scripts/fetch_secret.sh gcp-credentials.json > gcp-credentials.json
     - export GOOGLE_APPLICATION_CREDENTIALS=$(pwd -P)/gcp-credentials.json
     - sed -i -e s/dss.data.humancellatlas.org/dss.$DSS_TEST_STAGE.data.humancellatlas.org/ hca/default_config.json
-    - make install
   script:
     - make test
   except:
     - tags
 
+.windows-test-base:
+  stage: test
+  before_script:
+    #currently 3.6.8 is offered by the windows-runner
```

Comment:
I understand that we will have more explicit control of the python version(s) used for testing when we move to using Docker images with the GitLab Windows Runner Helper. In the interim, please at least include the actual Python version in the build output. Better still would be to verify that the Python version being used is the expected version (e.g. 3.6) and fail the build if it is not.

Reply:
Implemented print + check